components/p3a/p3a_metrics.h

```diff
@@ -0,0 +1,144 @@
+/* Copyright 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_P3A_P3A_METRICS_H_
+#define BRAVE_COMPONENTS_P3A_P3A_METRICS_H_
+
+namespace brave::p3a {
+
+// Allowlist for histograms that we collect.
+// A metric must be listed here to be reported.
+//
+// TODO(iefremov): Provide moar histograms!
+// Could be replaced with something built dynamically,
+// although so far we've found it a useful review point.
+//
+// clang-format off
+constexpr inline const char* kCollectedHistograms[] = {
```

Comment:
inline is redundant or i'm missing something?

Reply:
Redundant with constexpr? I was thinking it would protect against code bloat by telling the C++17 linker to de-duplicate the list if it ends up included in more than one place, but I didn't check whether this was necessary or effective.